tests/trainer/test_trainer.py

```diff
@@ -989,6 +990,40 @@ def test_bnb_compile(self):
             with self.assertRaises(ValueError):
                 _ = Trainer(tiny_model, args, train_dataset=train_dataset)  # noqa
 
+
+    @require_peft
+    @require_bitsandbytes
+    def test_peft_submodule(self):
+        from peft import LoraConfig, get_peft_model
+
+        # Simply tests if initializing a Trainer with a PEFT on a submodule will pass _is_peft_model check.
+        # Throws error when quantised but not recognised as a peft model.
```

Comment:
Could you clarify this comment a bit? I can't parse `"Throws error when quantised but not recognised as a peft model."`. Specifically, the "but" formulation makes it unclear the intended behaviour and causation (is it throwing an error because it's quantized? Should it throw an error when quantized? Should it recognize it's a peft model?)

Reply:
"but" is meant to be "as"